test/functional/p2p_invalid_messages.py

```diff
@@ -42,13 +48,33 @@ def set_test_params(self):
         self.setup_clean_chain = True
 
     def run_test(self):
+        self.test_buffer()
```

Comment:
This would allow not changing `test_large_inv()`:
```diff
     def run_test(self):
-        self.test_buffer()
         self.test_magic_bytes()
         self.test_checksum()
         self.test_size()
         self.test_msgtype()
         self.test_large_inv()
+        self.test_buffer()
         self.test_resource_exhaustion()
```